lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.html

```diff
@@ -43,7 +43,7 @@
         </adf-search-panel>
         <div class="adf-content-node-selector-document-list-container">
     <adf-toolbar>
-        <adf-toolbar-title [adf-node-counter]="getSelectedCount()">
+        <adf-toolbar-title>
```

Comment:
do we show the counter in 2 places? don't get why we have it in the panel and in the node selector container